cllc-public-app/ClientApp/src/app/app.component.html

```diff
@@ -1,7 +1,9 @@
-<div *ngIf="featureFlagService.initialized">
-  <section class="container not-supported-browser" *ngIf="isIE()">
-    <p>Internet Explorer is not supported</p>
-    <p>Please use Google Chrome, Firefox or Microsoft Edge</p>
+<div *ngIf="featureFlagService.initialized" *ngIf="isIE()">
+  <section class="container not-supported-browser">
+    <p><i class="fab fa-internet-explorer" style="margin-right:10px;"></i>Internet Explorer is no longer supported.</p>
+    <p>Please use <a href="https://google.com/chrome" target="_blank" class="IE">Google Chrome</a>, 
+      <a href="https://www.mozilla.org/en-CA/firefox/new/" target="_blank" class="IE">Firefox</a> or 
+      <a href="https://www.microsoft.com/en-us/edge" target="_blank" class="IE">Microsoft Edge</a>.</p>
   </section>
 
   <div class="app-outer" *ngIf="!isIE()">

```

Comment:
It looks like this div will never be visible as  its parent has an opposing ngif